src/komponenter/kalkulator/Fraværsperioder/Fraværsperioder.tsx

```diff
@@ -55,63 +49,19 @@ const Fraværsperioder: FunctionComponent<Props> = (props) => {
         );
     };
 
-    const oppdaterDatoIntervall = (
-        indeks: number,
-        datoIntervall: Partial<DatoIntervall>
-    ) => {
-        if (
-            datoIntervall.datoFra &&
-            !fraværInngårIPermitteringsperioder(
-                props.allePermitteringerOgFraværesPerioder.permitteringer,
-                datoIntervall
-            )
-        ) {
-            setAdvarselFraværsperiodeUtenforPermittering(
-                'Fravær som ikke inngår i permitteringsperioder telles ikke med i beregningen'
-            );
-        } else {
-            setAdvarselFraværsperiodeUtenforPermittering('');
-        }
-        const kopiAvFraværsperioder = [
-            ...props.allePermitteringerOgFraværesPerioder.andreFraværsperioder,
-        ];
-        kopiAvFraværsperioder[indeks] = datoIntervall;
-        props.setAllePermitteringerOgFraværesPerioder({
-            ...props.allePermitteringerOgFraværesPerioder,
-            andreFraværsperioder: kopiAvFraværsperioder,
-        });
-    };
-
     const fraVærsperiodeElementer = props.allePermitteringerOgFraværesPerioder.andreFraværsperioder.map(
         (fraværsintervall, indeks) => {
             return (
-                <DatoIntervallInput
```

Comment:
Flott å flytte dette til egen komponent 👍

Reply:
Jeg har et ønske, siden vi gjør denne flyttingen. Kan vi sende med ett fraværsintervall som props, sammen med slette- og endrefunksjon, i stedet for å sende med hele staten?